Cargo.toml

```diff
@@ -23,13 +23,12 @@ exclude = [
 license = "MPL-2.0"
 edition = "2018"
 
-
 [dependencies]
 merlin = "2.0.0"
 rand = "0.7.2"
 rand_core = { version = "0.5", default-features = false }
 # Built by default with "std", "alloc", "pairing", "groups" and "endo" features.
-dusk-bls12_381 = "0.1.0"
+dusk-bls12_381 = {git = "https://github.com/dusk-network/bls12_381", tag = "0.1.2"}
```

Comment:
We didn't push a new version of Bls since we were waiting for the ECC gate to then publish a new version of plonk with the new version of bls all at the same time.

Probably as you say it's better to just wait and use tags and releases until we define a good process to use crates.io versioning.

Reply:
What I said previously was to avoid publish on crates.io in the first place, during the development process.
Once is published is better continue to use it otherwise we add another variable to the whole mess.

Reply:
It is now referencing the crates.io version of BLS.